package.json

```diff
@@ -185,11 +186,24 @@
     "@testing-library/react": "16.0.0",
     "@testing-library/user-event": "14.5.2",
     "@types/node": "^20.14.9",
+    "@typescript-eslint/eslint-plugin": "^8.8.1",
+    "@typescript-eslint/parser": "^8.8.1",
     "babel-gettext-extractor": "^4.1.3",
     "babel-jest": "29.7.0",
     "benchmark": "^2.1.4",
     "eslint": "8.57.1",
-    "eslint-config-sentry-app": "2.9.0",
+    "eslint-config-sentry": "^2.10.0",
```

Comment:
can we remove this now?

Reply:
yup! the last step.

we do still need the plugin though, just not this config.

Reply:
ah right. i can't include https://github.com/getsentry/sentry/pull/80970/commits/00fae014631dc8a9eb67cf905bcb39076564b335 because that causes the sentry + getsentry dependency to break. I can land everything else then remove that package in more PRs